insightful/settings.py

```diff
@@ -0,0 +1,155 @@
+"""
+Django settings for insightful project.
+
+Generated by 'django-admin startproject' using Django 4.2.13.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.2/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.2/ref/settings/
+"""
+from pathlib import Path
+
+from decouple import config
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.2/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = config("SECRET_KEY")
```

Comment:
this isn't added in example env file

Reply:
I  have added this secret key in third line of this file. SECRET_KEY='your_project_secret_key'